src/OrchardCore/OrchardCore.ContentManagement.Abstractions/Utilities/StringExtensions.cs

```diff
@@ -180,44 +190,7 @@ public static string ToSafeName(this string name)
             return name;
         }
 
-        private static readonly HashSet<string> _reservedNames = new(StringComparer.OrdinalIgnoreCase)
-        {
-            nameof(ContentItem.Id),
-            nameof(ContentItem.ContentItemId),
-            nameof(ContentItem.ContentItemVersionId),
-            nameof(ContentItem.ContentType),
-            nameof(ContentItem.Published),
-            nameof(ContentItem.Latest),
-            nameof(ContentItem.ModifiedUtc),
-            nameof(ContentItem.PublishedUtc),
-            nameof(ContentItem.CreatedUtc),
-            nameof(ContentItem.Owner),
-            nameof(ContentItem.Author),
-            nameof(ContentItem.DisplayText),
-        };
-
-        public static bool IsReservedContentName(this string name)
-        {
-            if (_reservedNames.Contains(name))
-            {
-                return true;
-            }
-
-            return false;
-        }
-
-        /// <summary>
-        /// Whether the char is a letter between A and Z or not.
-        /// </summary>
-        public static bool IsLetter(this char c)
```

Comment:
mark these 2 methods obsolete and keep them just so we don't break anyone. you can change the logic to return `char.IsLetter(c)` and `chat.IsWhitespace()`